nntrainer/graph/network_graph.h

```diff
@@ -23,6 +23,7 @@
 #include <vector>
 
 #include <layer_internal.h>
+#include <loss_layer.h>
```

Comment:
is this intended?

Reply:
`int addLossLayer(const LossType loss_type);` requires LossType definition, I think we should move loss type definition to other place later.

Reply:
yes.

Reply:
Added here to track https://github.com/nnstreamer/nntrainer/issues/1063#issuecomment-813175687